stdpopsim/engines.py

```diff
@@ -118,11 +118,28 @@ def _warn_mutation_rate_mismatch(self, contig, demographic_model):
             demographic_model.mutation_rate, contig.mutation_rate
         ):
             warnings.warn(
-                "The demographic model has mutation rate "
+                "The demographic model has mean mutation rate "
```

Comment:
Demographic models (and, contigs) can't have mutation rate maps, so this is a bit misleading?
```suggestion
                "The demographic model has mutation rate "
```

Reply:
Noted. I was following how the code is setup and agree this was confusing/misleading me as well ...